library/src/scala/quoted/Quotes.scala

```diff
@@ -4229,6 +4229,13 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       /** Is this symbol `abstract` */
       def Abstract: Flags
 
+      /** Is this symbol is `abstract override`
+       *
+       *  A trait that has only abstract methods as members
+       *  and therefore can be represented by a Java interface.
```

Comment:
That description looks incorrect `abstract override` is a specific concept in Scala that is unrelated to Java interfaces: https://www.scala-lang.org/files/archive/spec/2.13/05-classes-and-objects.html#abstract-override

Reply:
I aligned the doc with https://github.com/lampepfl/dotty/blob/main/compiler/src/dotty/tools/dotc/core/Flags.scala#L282-L288

Reply:
This part of the documentation comment is for PureInterface, not for AbsOverride

Reply:
I updated the docs.

Reply:
I noticed that `AbsOverride` is pickled as `Abstract | Override` and then unpickled as `AbsOverride`. Maybe we should not have this flag in the reflection interface and handle `AbsOverride` as if both `Abstract` and `Override`.

Reply:
> I noticed that AbsOverride is pickled as Abstract | Override and then unpickled as AbsOverride. Maybe we should not have this flag in the reflection interface and handle AbsOverride as if both Abstract and Override.

I'm not sure diverging from the compiler implementation is a good idea here. In the specification, `abstract override` is a separate concept from `abstract` and `override`, it just reuses existing keywords. The tasty serialization format also reuses the flags bits, but that's just an optimization.